finality-aleph/src/network/manager/discovery.rs

```diff
@@ -472,9 +318,9 @@ mod tests {
         );
         assert_eq!(commands.len(), 1);
         assert!(matches!(&commands[0], (
-                DiscoveryMessage::Authentications(authentications),
+                DiscoveryMessage::Authentication(authentication),
                 DataCommand::SendTo(_, _),
-            ) if authentications == &vec![handler.authentication().unwrap()]));
+            ) if *authentication == handler.authentication().unwrap()));
```

Comment:
I was wondering, do we need to dereference `authentication`? Its type is `Authentication`, which is a tuple